Plugins/ActSVG/include/Acts/Plugins/ActSVG/IndexedSurfacesSvgConverter.hpp

```diff
@@ -274,14 +298,21 @@ static inline actsvg::svg::object xy(const ProtoIndexedSurfaceGrid& pIndexGrid,
     binText.push_back("Source:");
     binText.push_back(pGrid._bin_ids[ig]);
     binText.push_back("Target:");
-    for (const auto& sis : pIndices[ig]) {
-      binText.push_back(std::string("- object: ") + std::to_string(sis));
+    for (const auto [is, sis] : enumerate(pIndices[ig])) {
```

Comment:
Can we add an explicit namespace in front of ```enumerate```, so that it won't be confused with ```std::enumerate``` at some point?

Reply:
I think we should !